near/kycdao-ntnft/src/lib.rs

```diff
@@ -226,15 +359,28 @@ impl KycdaoNTNFT {
         }
     }
 
-    // TODO tokenVerificationURI???
+    /// @dev Returns the amount in NATIVE (yoctoNEAR) which is expected for a given amount of subscription time in seconds
+    /// @param seconds The number of seconds of subscription time to calculate the cost for
+    pub fn get_required_mint_cost_for_seconds(&self, seconds: u32) -> U128 {
+        U128(self.get_required_mint_cost_for_seconds_internal(seconds))
+    }
+
+    /// Returns the cost for subscription per year in USD, to SUBSCRIPTION_COST_DECIMALS decimal places
+    pub fn get_subscription_cost_per_year_usd(&self) -> u32 {
+        self.subscription_cost_per_year
+    }
+
+    pub fn get_mint_authorizer(&self) -> AccountId {
+        self.mint_authorizer.clone()
+    }
 
     /*****************
-    Admin
+    Mint authorizer functions
     *****************/
-    pub fn set_revoke_token(&mut self, token_id: TokenId, is_revoked: bool) {
+    pub fn set_verified_token(&mut self, token_id: TokenId, verified: bool) {
```

Comment:
Hmm, same deal on the setters, does it make sense to store this for a token that doesn't exist?